packages/-ember-data/tests/integration/relationships/belongs-to-test.js

```diff
@@ -242,9 +242,6 @@ module('integration/relationship/belongs_to Belongs-To Relationships', function(
       section: Section,
     });
 
-    env.registry.optionsForType('serializer', { singleton: false });
-    env.registry.optionsForType('adapter', { singleton: false });
```

Comment:
What was this actually doing?

Reply:
nothing, it was redundant as the helper already did exactly this (And did it twice lol)